src/components/app/components/banner.js

```diff
@@ -0,0 +1,61 @@
+                                <div style={{borderRadius:'15px', width: '50px', height: '50px', backgroundColor: 'white', margin: 'auto',}}>
+                                    <i style={{ color: '#106021', fontSize: '25px', paddingTop: '15px', }} className="fa fa-usd"></i>
+                                </div>
+                                <h3 style={{color: 'white',}}>Price Minimization</h3>
+                                <h5 style={{ color: 'white',}}><p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+                                
+                                </p></h5>
+                            </div>
+
+                            <div className="box" style={{width: '200px', display: 'block', textAlign: 'center',  padding: '15px',}}>
+                                <div style={{borderRadius:'15px', width: '50px', height: '50px', backgroundColor: 'white', margin: 'auto',}}>
+                                    <i style={{ color: '#106021', fontSize: '20px', paddingTop: '15px', }} className="fa fa-trash"></i>
+                                </div>
+                                <h3 style={{color: 'white',}}>Reduce Food Wastage</h3>
+                                <h5 style={{ color: 'white',}}><p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+                                
+                                </p></h5>
+                            </div>
+
+                            <div className="box" style={{width: '200px', display: 'block', textAlign: 'center',  padding: '15px',}}>
+                                <div style={{borderRadius:'15px', width: '50px', height: '50px', backgroundColor: 'white', margin: 'auto',}}>
+                                     <i style={{ color: '#106021', fontSize: '20px', paddingTop: '15px', }} className="fa fa-users"></i>
+                                </div>
+                                <h3 style={{color: 'white',}}>Social Economic Driven</h3>
+                                <h5 style={{ color: 'white',}}><p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+                                
+                                </p></h5>
+                            </div>
+                        </div>
+                        <Button>Learn More</Button>
+                    </div>
+        );
+    }
+}
```

Comment:
@EngrEric can you show @mcbobby7 how to store all the information in the an array of objects and how to map over that array to display the content